static/src/javascripts/projects/common/views/international-message.html

```diff
@@ -1,9 +1,9 @@
 <p class="site-message__message" id="site-message__message">
-    You’re viewing a beta release of the Guardian’s international edition.
+    You are viewing the Guardian’s international edition.
 </p>
 <ul class="site-message__actions u-unstyled">
     <li class="site-message__actions__item">
         <i class="i i-back"></i>
-        <a class="js-main-site-link" rel="nofollow" href="/preference/edition/uk" data-link-name="opt-out">Opt out and go to UK edition</a>
+        <a class="js-main-site-link" rel="nofollow" href="/preference/edition/uk" data-link-name="opt-out">Opt out and visit the UK edition instead</a>
```

Comment:
it is an opt out, or just a difference?  I'd consider saying simply "Visit the UK edition instead"